src/ingestors/fxhash/abi.ts

```diff
@@ -1,125 +1,4 @@
 export const FXHASH_BASE_FRAME_ABI = [
-  {
-    inputs: [
-      { internalType: 'address', name: '_owner', type: 'address' },
-      { internalType: 'address', name: '_controller', type: 'address' },
-    ],
-    stateMutability: 'nonpayable',
-    type: 'constructor',
-  },
-  { inputs: [], name: 'AddressZero', type: 'error' },
```

Comment:
We may want to keep the errors - this _should_ make them more readable if our mint/simulation fails since they appear to be using native Solidity errors vs reverting with a string.